### PR TITLE
adaptative cost-sensitive online classification

### PR DESCRIPTION
**Pull request by CharlesAuthier**

### CODE REVIEW

File: machine-learning/_posts/2018-04-26-cost-sensitive-OC [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+author: "Charles Authier"
+pdf: "https://arxiv.org/pdf/1804.02246"
+cite:
+    authors: "Peilin Zhao, Yifan Zhang, Min Wu, Steven C. H. Hoi, Mingkui Tan, Junzhou Huang"
+    title:   "Adaptive Cost-sensitive Online Classification"
+    venue:   "https://arxiv.org/pdf/1804.02246"
+---
+
+## Intro
+They proposed fiew empirically algorithms and validate their effectiveness and properties.
```
Comment (Dref360) [inline machine-learning/_posts/2018-04-26-cost-sensitive-OC]: fiew?

**Comment by lemairecarl** [comment]
Why isn't your file a `.markdown` or `.md`

File: machine-learning/_posts/2018-04-26-cost-sensitive-OC.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+author: "Charles Authier"
+pdf: "https://arxiv.org/pdf/1804.02246"
+cite:
+    authors: "Peilin Zhao, Yifan Zhang, Min Wu, Steven C. H. Hoi, Mingkui Tan, Junzhou Huang"
+    title:   "Adaptive Cost-sensitive Online Classification"
+    venue:   "https://arxiv.org/pdf/1804.02246"
+---
+
+## Intro
+They proposed three empirically algorithms and validate their effectiveness and properties.
```
Comment (jodoin) [inline machine-learning/_posts/2018-04-26-cost-sensitive-OC.md]: precise what those algorithms do.  These are gradient descent optimization algorithms?

File: machine-learning/_posts/2018-04-26-cost-sensitive-OC.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+author: "Charles Authier"
+pdf: "https://arxiv.org/pdf/1804.02246"
+cite:
+    authors: "Peilin Zhao, Yifan Zhang, Min Wu, Steven C. H. Hoi, Mingkui Tan, Junzhou Huang"
+    title:   "Adaptive Cost-sensitive Online Classification"
+    venue:   "https://arxiv.org/pdf/1804.02246"
+---
+
+## Intro
+They proposed three algorithms and validate their effectiveness and properties for solving cost-sensitive online classification problems.
```
Comment (lemairecarl) [inline machine-learning/_posts/2018-04-26-cost-sensitive-OC.md]: Please give an example of a "cost-sensitive online classification problem"

File: machine-learning/_posts/2018-04-26-cost-sensitive-OC.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+They approximate the second covariance matrix Σ by a small number of carefully selective directions.
```
Comment (lemairecarl) [inline machine-learning/_posts/2018-04-26-cost-sensitive-OC.md]: "second covariance matrix Σ" ? You have to give more context. So there's a *first* covariance matrix?

**Pull request merged**